src/index.html

```diff
@@ -9,10 +9,136 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The stylesheet is linked with a `.scss` extension. HTML cannot directly use SASS files. You need to compile the SASS file into a CSS file and link the resulting `.css` file here.